ss/errors.go

```diff
@@ -0,0 +1,13 @@
+package ss
+
+import (
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// StoreCodespace defines the store package's unique error code space.
+const StoreCodespace = "store"
+
+var (
+	ErrKeyEmpty      = sdkerrors.Register(StoreCodespace, 1, "key empty")
```

Comment:
I wonder if we can just create a new error instead of depending on the cosmos sdk error?

Reply:
Sorry what do you mean? We are creating a new error here

Reply:
Updated